client/fingerprint/host_test.go

```diff
@@ -35,8 +36,17 @@ func TestHostFingerprint(t *testing.T) {
 		t.Fatalf("should generate a diff of node attributes")
 	}
 
+	commonAttributes := []string{"os.name", "unique.hostname", "kernel.name"}
+	nonWindowsAttributes := append(commonAttributes, "os.version")
+	windowsAttributes := append(commonAttributes, "os.build")
```

Comment:
With your most recent change `os.version` should appear for both:

```suggestion
	commonAttributes := []string{"os.name", "os.version", "unique.hostname", "kernel.name"}
	nonWindowsAttributes := append(commonAttributes, "kernel.version")
	windowsAttributes := append(commonAttributes, "os.build")
```